chain-signatures/contract/tests/tests.rs

```diff
@@ -251,7 +251,7 @@ async fn test_contract_sign_request() -> anyhow::Result<()> {
     let err = sign_and_validate(&request, None, &contract)
         .await
         .expect_err("should have failed with timeout");
-    assert!(err.to_string().contains("Signature has timed out"));
+    assert!(err.to_string().contains("Signature request has timed out"));
```

Comment:
we should be able to do `contains(Error.to_string())` since they implement `Display`